src/galaxy_mergers.cpp

```diff
@@ -128,7 +145,57 @@ void GalaxyMergers::merging_timescale(Galaxy &galaxy, SubhaloPtr &primary, Subha
 		double mp, double tau_dyn, int snapshot, bool transfer_types2)
 {
 	// Define merging timescale and redefine type of galaxy.
-	if(parameters.tau_delay > 0){
+	if(parameters.model==GalaxyMergerParameters::POULTON20){
+
+		// Find the objects physical position
+		double conversion_factor = cosmo_params.Hubble_h * (1 +  simparams.redshifts[snapshot]);
+		double xrel = (secondary->position.x - primary->position.x) * conversion_factor;
+		double yrel = (secondary->position.y - primary->position.y) * conversion_factor;
+		double zrel = (secondary->position.z - primary->position.z) * conversion_factor;
+		double r = sqrt(xrel*xrel + yrel*yrel + zrel*zrel);
```

Comment:
I think you could do `double r = ((secondary->position - primary->position) * conversion_factor).norm()` instead of the per-element computation, makes the code more compact and conveys the purpose more clearly. Same below.

Reply:
I just realised I changed these but all the components are required in the calculation for the angular momentum. Should I revert to my initial commit and I will just correct the above spelling?

Reply:
Sure, I hadn't realised about that either, sorry for the extra confusion!